### PR TITLE
AddingTwo Numbers  Problem Works fine

### PR DESCRIPTION
**Pull request by noor848**
image

### CODE REVIEW

File: Prob3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+ * Definition for singly-linked list.
+ * public class ListNode {
+ *     public int val;
+ *     public ListNode next;
+ *     public ListNode(int val=0, ListNode next=null) {
+ *         this.val = val;
+ *         this.next = next;
+ *     }
+ * }
+ */
```
Comment (hayatzsala) [inline Prob3/Program.cs]: [Clean Code] in general avoid put any unused code in comments

File: Prob3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                        l1.next = new ListNode(raise);
+                        break;
+
+                    }
+
+
+                    l1 = l1.next;
+
+                }
+
```
Comment (hayatzsala) [inline Prob3/Program.cs]: empty lines 🤫

File: Problem.sln [hunk 3]
```diff
@@ -0,0 +1,31 @@
+﻿
```
Comment (hayatzsala) [inline Problem.sln]: check that there is not any reference to Project3, so it doesn't appear on the solution

Comment (hayatzsala) [reply Problem.sln]: just by adding the project into the solution

File: Prob3/Program.cs [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+ *     public int val;
+ *     public ListNode next;
+ *     public ListNode(int val=0, ListNode next=null) {
+ *         this.val = val;
+ *         this.next = next;
+ *     }
+ * }
+ */
+public class Solution
+{
```
Comment (hayatzsala) [inline Prob3/Program.cs]: @noor848 Where is the` Main` method?? 
and where is the` ListNode` class?? (remove the comment above)

File: Prob3/Program.cs [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+            l3.next = new ListNode(raise);
+            l3 = l3.next;
+
+        }
+
+        return l4.next;
+
+
+
+    }
```
Comment (hayatzsala) [inline Prob3/Program.cs:113]: @noor848 you get the right logic, Good Job 👏

Comment (noor848) [reply Prob3/Program.cs:113]: Thanks God 😪😂

File: Prob3/Program.cs (hunk 4 above)
Comment (noor848) [reply Prob3/Program.cs]: fixed

**Pull request merged**